src/pages/workspaces/workspace/analysis/runtime-common.js

```diff
@@ -144,7 +144,6 @@ export const SaveFilesHelpRStudio = () => {
         ...Utils.newTabLinkProps
       }, ['move them to the workspace bucket.'])
     ]),
```

Comment:
does this `,` need to be removed?

Reply:
oh good catch, it worked fine with it, but I can remove that as well to keep things clean